code/game/machinery/vending/cm_vending.dm

```diff
@@ -1063,10 +1067,12 @@ GLOBAL_LIST_INIT(cm_vending_gear_corresponding_types_list, list(
 		var/priority = myprod[priority_index]
 
 		var/obj/item/I = item_ref
```

Comment:
Why not just have
`var/item_ref = myprod[3]`
be 
`var/obj/item/item_ref = myprod[3]`
since you've already proven that it cast to var/obj/item/I can pass the islist test. That'll also mean you don't have a single character var too.

Reply:
well that was existing code so i rolled with it but that's very goofy indeed, will change